modules/ROOT/pages/java/java-invoke-method.adoc

```diff
@@ -348,8 +362,8 @@ INFO  2019-02-22 09:21:32 ... [event: 4c31f...] ... LoggerMessageProcessor: 7
 
 == Invoke Methods with DataWeave
 
-You can invoke Java methods using DataWeave expressions. This practice is
-especially helpful for methods that return `boolean` values.
+You can also use the `invoke` operation of the Java Module using DataWeave expressions.
```

Comment:
explain how to use the function, which parameter it takes, etc, not just an example

Reply:
Done